crates/router/src/core/payment_link.rs

```diff
@@ -64,6 +65,7 @@ pub async fn form_payment_link_data(
     key_store: domain::MerchantKeyStore,
     merchant_id: common_utils::id_type::MerchantId,
     payment_id: String,
+    locale: Option<String>,
```

Comment:
this could be an enum for validation

Reply:
We don't want to throw an error incase if client passes invalid enum of accept headers, that's the reason it was Option<String>.